src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsWriteEvent.cs

```diff
@@ -647,6 +652,8 @@ private void Test_WriteEvent_ByteArray(bool useSelfDescribingEvents, Listener li
                         Assert.Equal(1000, (long)evt.PayloadValue(1, "lng"));
                     }));
 
+                /* TODO: NULL byte array does not seem to be supported.
+                 * An EventSourceMessage event is written for this case.
```

Comment:
Is this https://github.com/dotnet/coreclr/pull/1988?

Reply:
Close, but it doesn't look like this matches one of the overloads that was modified in that PR.  I suspect we need to modify the params overload to account for this one.